events.xml.append

```diff
@@ -38369,7 +38369,7 @@ IDEAS
 	<text planet="PLANET_UNPOPULATED">A legendary breed of changer lizards inhabits this world. You scanners uncover them while posing as a heard of sheep-like cattle. The crew captures one of the shape shifter animals an brings it aboard.</text>
 	<text>The Curators data was right: A small portion of space whale ambra is found in this system. The rare substance is left behind by the space inhabiting behemoth once in a decade. It is said to stimulate telepathic abilities.</text>
```

Comment:
<text>The Curator's data was right: A small portion of space whale ambra is found in this system. The rare substance is left behind by the space inhabiting behemoth once in a decade. It is said to stimulate telepathic abilities.</text>